### PR TITLE
Added the entry to select multiple models and verified the dockerfile

### PR DESCRIPTION
**Pull request by zanghao2**
for issue #148 https://github.com/AICoE/prometheus-anomaly-detector/issues/148

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -3,10 +3,12 @@ name: prophet-env
 channels:
   - conda-forge
 dependencies:
-  - python=3.8
+  - python=3.6
```
Comment (4n4nd) [inline environment.yml:6]: @zanghao2 any reason the python version is being changed to `3.6`?

**Review by 4n4nd — CHANGES_REQUESTED** [review]
Thanks for the PR!. I have wanted to add this feature for a really long time.

I have asked a question about the dependencies ^.

Could you also update the README and add the `FLT_MODEL_TYPE` flag in it?

**Comment by sasah** [comment]
https://github.com/tensorflow/models/issues/9706#issuecomment-868542761

**Pull request closed without merging**